src/log-httplog.c

```diff
@@ -183,80 +183,73 @@ static void LogHttpLogCustom(LogHttpLogThread *aft, htp_tx_t *tx, const SCTime_t
                 break;
             case LOG_HTTP_CF_REQUEST_METHOD:
             /* METHOD */
-                if (tx->request_method != NULL) {
-                    PrintRawUriBuf((char *)aft->buffer->buffer, &aft->buffer->offset,
-                                aft->buffer->size, (uint8_t *)bstr_ptr(tx->request_method),
-                                bstr_len(tx->request_method));
-                } else {
-                    MemBufferWriteString(aft->buffer, LOG_CF_NONE);
-                }
+            if (htp_tx_request_method(tx) != NULL) {
+                PrintRawUriBuf((char *)aft->buffer->buffer, &aft->buffer->offset, aft->buffer->size,
+                        (uint8_t *)bstr_ptr(htp_tx_request_method(tx)),
+                        bstr_len(htp_tx_request_method(tx)));
+            } else {
+                MemBufferWriteString(aft->buffer, LOG_CF_NONE);
+            }
                 break;
             case LOG_HTTP_CF_REQUEST_URI:
             /* URI */
-                if (tx->request_uri != NULL) {
-                    datalen = node->maxlen;
-                    if (datalen == 0 || datalen > bstr_len(tx->request_uri)) {
-                        datalen = bstr_len(tx->request_uri);
-                    }
-                    PrintRawUriBuf((char *)aft->buffer->buffer, &aft->buffer->offset,
-                                aft->buffer->size, (uint8_t *)bstr_ptr(tx->request_uri),
-                                datalen);
-                } else {
-                    MemBufferWriteString(aft->buffer, LOG_CF_NONE);
+            if (htp_tx_request_uri(tx) != NULL) {
+                datalen = node->maxlen;
+                if (datalen == 0 || datalen > bstr_len(htp_tx_request_uri(tx))) {
+                    datalen = bstr_len(htp_tx_request_uri(tx));
```

Comment:
Did the indentation break here, considering the swtich-case statement?

Reply:
clang-format did it

Reply:
Ah! Just noticed that clang doesn't seem to know how to handle this whole switch-case statement. There are two different indentation styles going on... It's breaking our guidelines, in doing so: https://docs.suricata.io/en/latest/devguide/codebase/code-style.html#switch-statements 🤔